dapr/clients/grpc/_response.py

```diff
@@ -467,3 +467,89 @@ def __init__(
     def items(self) -> Sequence[BulkStateItem]:
         """Gets the items."""
         return self._items
+
+
+class ConfigurationItem:
+    """A config item from get_configuration API.
+
+    Attributes:
+        key (str): config's key.
+            value (Union[bytes, str]): config's value.
+            version (str): config's version.
+            metadata (str): metadata
+    """
+
+    def __init__(
+            self,
+            key: str,
+            value: str,
+            version: str,
+            metadata: Optional[Dict[str, str]] = dict()):
+        """Initializes ConfigurationItem item from :obj:`runtime_v1.ConfigurationItem`.
+
+        Args:
+            key (str): config's key.
+            value (str): config's value.
+            version (str): config's version.
+            metadata (Optional[Dict[str, str]] = dict()): metadata
+        """
+        self._key = key
+        self._value = value
+        self._version = version
+        self._metadata = metadata
+
+    def text(self) -> str:
+        """Gets content as str."""
+        return to_str(self._value)
+
+    def json(self) -> Dict[str, object]:
+        """Gets content as deserialized JSON dictionary."""
+        return json.loads(to_str(self._value))
```

Comment:
quick question: is the value a single field or consists multiple items?

Reply:
It consists of multiple items.